test/api_integration/apis/telemetry/telemetry_last_reported.ts

```diff
@@ -28,7 +28,7 @@ export default function optInTest({ getService }: FtrProviderContext) {
       await supertest
         .get('/api/telemetry/v2/last_reported')
         .set('kbn-xsrf', 'xxx')
-        .expect(200, { lastReported: undefined });
+        .expect(200, { lastReported: null });
```

Comment:
__note:__ justification, the test is broken in `8.6`, and upgraded `superagent` revealed that:
```http
GET /api/telemetry/v2/last_reported HTTP/1.1
Host: localhost:5620
Accept-Encoding: gzip, deflate
User-Agent: node-superagent/3.8.3

HTTP/1.1 200 OK
content-type: application/json; charset=utf-8

{"lastReported":null}
```